src/database/migrations/1.0.0/1696782681632-SystemServicesNode.ts

```diff
@@ -0,0 +1,85 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm";
+
+export class SystemServicesNode1696782681632 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'System')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'DHCP')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'Keepalived')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'HAProxy')"
+        );
+
+        const firewalls = await queryRunner.query(
+            "SELECT `id` FROM `fwc_tree` WHERE `node_type` = 'firewall'"
+        );
+
+        for (const firewall of firewalls) {
+            await queryRunner.query(
+                "INSERT INTO `fwc_tree` (`parent_id`, `name`, `node_type`) VALUES (?, 'System', 'SYS')",
+                [firewall.id]
+            );
+        }
+
+        const systems = await queryRunner.query(
+            "SELECT `id` FROM `fwc_tree` WHERE `name` = 'System'"
+        );
+
+        for (const system of systems) {
+            // Inserta nodo DHCP
+            await queryRunner.query(
+                "INSERT INTO `fwc_tree` (`parent_id`, `name`, `node_type`) VALUES (?, 'DHCP', 'SYS_CHILD')",
+                [system.id]
+            );
```

Comment:
What about oder important fields of the `fwc_tree` table ?
```mysql
mysql> desc fwc_tree;
+------------+--------------+------+-----+---------+----------------+
| Field      | Type         | Null | Key | Default | Extra          |
+------------+--------------+------+-----+---------+----------------+
| id         | int(11)      | NO   | PRI | NULL    | auto_increment |
| name       | varchar(255) | YES  |     | NULL    |                |
| id_parent  | int(11)      | YES  | MUL | NULL    |                |
| node_order | tinyint(2)   | NO   |     | 0       |                |
| node_type  | char(3)      | YES  |     | NULL    |                |
| id_obj     | int(11)      | YES  | MUL | NULL    |                |
| obj_type   | int(11)      | YES  | MUL | NULL    |                |
| fwcloud    | int(11)      | YES  | MUL | NULL    |                |
+------------+--------------+------+-----+---------+----------------+
8 rows in set (0.00 sec)
```

Reply:
And the id_obj field? It must contain the ID of the firewall or cluster to which this node belongs to.
CLUE: You can find it in the id_obj field of the firewall or cluster node.